graphql/execute.lua

```diff
@@ -313,7 +313,6 @@ end
 
```

Comment:
> This PR fixes the logic to collect all errors from all resolvers
> into a single map that can be returned to requester.

Nit: the word 'map' it was a bit confusing for me in this context (hower the test shows that it is actually an array).

Reply:
Fixed